packages/blade/src/components/Drawer/DrawerContext.ts

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
We should globally disable this rule.

Reply:
Need @anuraghazra approval 🙈 

We can create issue and discuss it there